src/NuGet.Clients/NuGet.PackageManagement.UI/Resources.resx

```diff
@@ -1005,4 +1005,20 @@ Please see https://aka.ms/troubleshoot_nuget_cache for more help.</value>
     <value>This package version is deprecated. Use {0} instead.</value>
     <comment>{0} is a NuGet package id (string)</comment>
   </data>
+  <data name="Billion" xml:space="preserve">
+    <value>{0:G3}B</value>
+    <comment>Billions in short format notation</comment>
+  </data>
+  <data name="Million" xml:space="preserve">
+    <value>{0:G3}M</value>
+    <comment>Millions in short format notation</comment>
+  </data>
+  <data name="Thousand" xml:space="preserve">
+    <value>{0:G3}K</value>
+    <comment>Thousands in short format notation</comment>
+  </data>
+  <data name="Trillion" xml:space="preserve">
+    <value>{0:G3}T</value>
+    <comment>Trillions in short format notation</comment>
```

Comment:
Please specify which trillions are you referring to:

```suggestion
    <comment>American trillions in short format notation</comment>
```

Reply:
Does EU and US trillions use different notations?

Reply:
https://en.wikipedia.org/wiki/Long_and_short_scale

Reply:
> https://en.wikipedia.org/wiki/Long_and_short_scale

Thanks you. Now I understand what `milliard` actually means, previously I always thought it's just another regional spelling for million.